lib/manager/cargo/extract.ts

```diff
@@ -19,13 +27,24 @@ function extractFromSection(
     let skipReason: SkipReason;
     let currentValue = sectionContent[depName];
     let nestedVersion = false;
+    let registryUrls: string[] = ['https://crates.io'];
```

Comment:
We prefer that default registry URLs are not inserted by the manager extract but are instead declared as the datasource default ([example](https://github.com/renovatebot/renovate/blob/1f2d9158df0f855212376b80f7366e3af2c7e98c/lib/datasource/docker/index.ts#L20) )

Reply:
Addressed, thanks!